app/build.gradle

```diff
@@ -1,17 +1,22 @@
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 
+// This file exists in root folder and it is a dummy file.
+// When releasing, replace it by the real key
 def keystorePropertiesFile = rootProject.file("keystore.properties")
+
 def keystoreProperties = new Properties()
 keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
 
 android {
     signingConfigs {
         config {
-            keyAlias keystoreProperties['keyAlias']
-            keyPassword keystoreProperties['keyPassword']
-            storeFile file(keystoreProperties['storeFile'])
-            storePassword keystoreProperties['storePassword']
+            if (keystorePropertiesFile.exists()) {
```

Comment:
Since you are creating a dummy keystore.properties, this check will always be true